utils/frame/benchmarking-cli/src/pallet/command.rs

```diff
@@ -364,13 +372,15 @@ impl PalletCmd {
 						if elapsed >= time::Duration::from_secs(5) {
 							timer = time::SystemTime::now();
 							log::info!(
-								"Running Benchmark: {}.{} {}/{} {}/{}",
+								"Running Benchmark: {}.{}({} args) {}/{} {}/{}",
 								String::from_utf8(pallet.clone())
 									.expect("Encoded from String; qed"),
 								String::from_utf8(extrinsic.clone())
 									.expect("Encoded from String; qed"),
+								components.len(),
 								s + 1, // s starts at 0. todo show step
-								self.steps,
+								// `steps` is the number of steps *per* component.
+								all_components.len(),
```

Comment:
is this TODO still needed?

Seems like it may be removed

also the comments references `steps`, but where?

Reply:
i think todo can be removed

Reply:
todo seems old, I couldn't figure out the context it was added even by looking at the old commit history

as for L382, it was introduced https://github.com/bernardoaraujor/substrate/commit/c029dc6216d39905f7a3e4c88947fb511694e337 what was the rationale there @ggwpez ?

Reply:
I meant `s`, not `steps` :man_facepalming:   
Will fix